kaggle_environments/envs/football/football.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "football",
+    "agents": [2],
+    "configuration": {
+      "episodeSteps": 30,
+      "scenario_name": {
+          "description": "Name of the scenario: for example 11_vs_11_stochastic. Look inside scenarios/ dir of GoogleResearchFootball.",
+          "type": "string",
+          "default": "11_vs_11_stochastic"
+      },
+      "team_1": {
+          "description": "Number of players from the first team that agent controls.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 10,
+          "default": 1
+      },
+      "team_2": {
+          "description": "Number of players that other agent controls. If set to 0 - the second agent will always have to return 0 as action.",
+          "type": "integer",
+          "minimum": 0,
+          "maximum": 10,
+          "default": 0
+      }
+    },
+    "reward": {
+      "description": "1.0 for scored goal, -1.0 for lost goal (it is given only when the goal is scored, and changes back to 0 afterwards)",
```

Comment:
It can depend on us.
The underlying environment is giving a +1, -1 reward only after the goal was scored (and then rewards go back to 0) - this makes training more stable.